package.json

```diff
@@ -23,7 +23,6 @@
     "preview": "bun ./src/scripts/preview/run.ts",
     "svgo": "svgo -i icons -o icons -q",
     "test": "bun test",
-    "version": "bun run changelog && git add CHANGELOG.md",
```

Comment:
Was this intentional?

Reply:
> Was this intentional?

This had to be removed because the `"version"` script uses the life cycle of the `npm version` command, and the `bun run changelog` command updates `CHANGELOG.md` and returns the release notes at the same time, so we don't like this option and that's why I deleted this script.